scripts/travis_install

```diff
@@ -14,13 +14,13 @@ rm -rf $MINICONDA.sh
 
 python -V
 
-PINNED_PKGS=$(cat <<EOF
-binstar ==0.10.1
-EOF
-)
-echo -e "$PINNED_PKGS" > /home/travis/miniconda/conda-meta/pinned
+# PINNED_PKGS=$(cat <<EOF
+# binstar ==0.10.1
+# EOF
+# )
+# echo -e "$PINNED_PKGS" > /home/travis/miniconda/conda-meta/pinned
```

Comment:
I agree, Aaron and I were just testing that it would work using anaconda. It might even want to go into another PR unless @damianavila is ok with keeping it in here.